.travis.yml

```diff
@@ -1,6 +1,7 @@
 # This provides an initial configuration to
 # test xp-framework on http://travis-ci.org/
 
+sudo: false
```

Comment:
This disables `sudo` permissions on the travis buildbox (which the build doesn't use currently), therefore enables use of containerized boxes (using Docker) and thus results in faster build times.